website/content/en/docs/kubebuilder/operator-scope.md

```diff
@@ -0,0 +1,298 @@
+
+mgr, err = ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    NewCache:           cache.MultiNamespacedCacheBuilder(namespaces),
+    MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
+})
+...
+```
+
+In the above example, if a CR(CustomResource), which is managed by its controllers, is applied to a namespace that 
+is not in this List of `namespaces` then, its controller will not re-trigger its reconcile since the namespaced is NOT 
+part of the [Manager][ctrl-manager] scope.
+
+**IMPORTANT:** Note that this is not intended to be used for excluding namespaces, this is better done via a Predicate. 
+Also note that you may face performance issues when using this with a high number of namespaces.
+
+## Restricting Roles and permissions
+
+The operator-scope defines the [Manager][ctrl-manager] cache scope but not the permissions to access the resources. After updating the Manager's scope to be Namespaced, the [Kubernetes Role-Based Access Control or the (RBAC)][k8s-rbac] permissions should be restricted accordingly.
+
+In the projects, the permissions are specified in the directory `config/rbac/`. The RBAC manifest at `config/rbac/role.yaml` and `config/rbac/role_binding.yaml` specifies the permissions needed by the operator to manage its resources, and is a `ClusterRole` by default. 
+ 
+**NOTE** The files `auth_proxy_client_clusterrole.yaml`, `auth_proxy_role.yaml`, `auth_proxy_role_binding.yaml` are also 
+defined to allow permissions across the cluster as well and are used by the operator. More info see: 
+[kube-rbac-proxy][kube-rbac-proxy]. Also, it is important to highlight that it is an alpha feature that is not officially 
+supported and can be disabled by commenting the lines in the config `config/rbac/kustomization.yaml` and in the 
+`config/default/kustomization.yaml`. See:
+
+-  In `config/rbac/kustomization.yaml`:
+```yaml
+...
+# Comment the following 4 lines if you want to disable
+# the auth proxy (https://github.com/brancz/kube-rbac-proxy)
+# which protects your /metrics endpoint.
+- auth_proxy_service.yaml
+- auth_proxy_role.yaml
+- auth_proxy_role_binding.yaml
+- auth_proxy_client_clusterrole.yaml
+```
+
+- In `config/default/kustomization.yaml`:
+```yaml
+...
+patchesStrategicMerge:
+  # Protect the /metrics endpoint by putting it behind auth.
+  # If you want your controller-manager to expose the /metrics
+  # endpoint w/o any authn/z, please comment the following line.
+- manager_auth_proxy_patch.yaml
+...
+```
+
+And then, see that for each API (CRD) scaffolded a specific role file with permissions to `edit` or `view` the Kind 
+resource in the cluster is created too. It is not used by the project, but are helpers to allow you to perform specific 
+configurations. For example, after creating a new API(CRD) with the Memcached Kind, we will able to see the roles 
+`memcached_editor_role.ymal` and `memcached_viewer_role.yaml` created there.
```

Comment:
Grammar and typos. 
Also we can rephrase this to be a lot more concise as well.
```suggestion
The RBAC manifests with the names `<kind>_editor_role.yaml` and `<kind>_viewer_role.yaml` are meant for end users or clients to view and edit CustomResources for that Kind.
```

Reply:
Addressed with small changes.